rpcs3/Emu/System.cpp

```diff
@@ -368,9 +368,22 @@ void Emulator::Init(bool add_only)
 	// Disable incompatible settings
 	fixup_ppu_settings();
 
+	bool must_create_inside_emu_dir = false;
```

Comment:
please just use a second param. this is very confusing

Reply:
Fixed.